modules/sr/robot3/arduino.py

```diff
@@ -0,0 +1,289 @@
+"""The Arduino module provides an interface to the simulated Arduino."""
+from __future__ import annotations
+
+import logging
+from enum import Enum, IntEnum
+from types import MappingProxyType
+
+from controller import Robot
+from sr.robot3.utils import BoardIdentity
+from sr.robot3.arduino_devices import (
+    Led,
+    EmptyPin,
+    PinDevice,
+    Microswitch,
+    DistanceSensor,
+    PressureSensor,
+)
+
+logger = logging.getLogger(__name__)
```

Comment:
This appears to be unused.

Reply:
813d119db0f894315bfc2ace5ab8e45c69a0dc9d